<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>391</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/5947114fd9b1f751753920dbebc69c55.jpg);</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>411</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #404040;
</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>341</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 9pt &quot;Stencil&quot;;
border-bottom:5px solid #1565C0;
color:#fff;
</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:28pt;&quot;&gt;Log In&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="username_edt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
padding-left:8px;
color:#fff;
background-color:#9E9E9E;
font-size:17px;</string>
     </property>
     <property name="placeholderText">
      <string>User Name</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="password_edt">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>220</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:5px;
padding-left:8px;
color:#fff;
background-color:#9E9E9E;
font-size:17px;</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="login_btn">
     <property name="geometry">
      <rect>
       <x>42</x>
       <y>370</y>
       <width>341</width>
       <height>51</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
border-radius:7px;
background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:0.994, stop:0 rgba(0, 80, 120, 255), stop:0.3125 rgba(14, 8, 73, 255), stop:0.548077 rgba(28, 17, 145, 255), stop:0.764423 rgba(85, 170, 255, 255));
color:#fff;
font: 12pt &quot;MS Reference Sans Serif&quot;;
 transition: 0.3s;
}
QPushButton:hover{
font-size:20px;
}

</string>
     </property>
     <property name="text">
      <string>Log in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>320</y>
       <width>171</width>
       <height>29</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color:#1565C0;
font-size:18px;
padding-right:7px;
font-weight:800;
background-color:  #404040;
border:none;}
QPushButton:hover{
color:#fff;
}</string>
     </property>
     <property name="text">
      <string>Forgot Password?</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>440</y>
       <width>171</width>
       <height>29</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
color:#1565C0;
font-size:18px;
padding-right:7px;
font-weight:800;
background-color: #404040;
border:none;
}
QPushButton:hover{
color:#fff;
}</string>
     </property>
     <property name="text">
      <string>Creat new account</string>
     </property>
    </widget>
    <widget class="QLabel" name="warning_lbl">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>290</y>
       <width>251</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">padding-left:7px;
color: rgb(253, 0, 0);
font: 9pt &quot;Zar&quot;;
font: 9pt &quot;Segoe UI&quot;;
text-decoration: underline;</string>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;UserName or Password Not found!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
